server/handlers/getSurvey.test.js

```diff
@@ -1,31 +1,20 @@
 /* eslint-disable no-undef */
-// const request = require('supertest');
-// const { initDb, closeDb } = require('../databaseConnection');
-// const app = require('../app');
+const request = require('supertest');
+const { initDb, closeDb } = require('../databaseConnection');
+const app = require('../app');
 
-// beforeEach(() => {
-//   return initDb();
-// });
-
-// afterEach(() => {
-//   return closeDb();
-// });
 
 describe('Testing GET /surveys/:id', () => {
-  it('Responds with JSON and status 200 when the survey has been found successfully.', (done) => {
-    done();
-    // const documentIdToFind = '507f1f77bcf86cd799439001';
-    // request(app)
-    //   .get(`/surveys/${documentIdToFind}`)
-    //   .set('Accept', 'application/json')
-    //   .expect(200)
-    //   .expect('Content-Type', /json/)
-    //   .end((err, res) => {
-    //     if (err) {
-    //       return done(err);
-    //     }
-    //     expect(res.body.auth).toEqual(true);
-    //     return done();
-    //   });
+  it('Responds with JSON and status 200 when the survey has been found successfully.', async () => {
+
+    await initDb();
```

Comment:
In some tests, we are calling initDb() and ending with closeDb(), in other tests, we are using beforeEach and afterEach. Not important as it works but something to consider for consistency.

Reply:
agreed there is some inconsistency, but given time pressures I'm not going to comb through on this level of granularity. Thanks for raising this though!